i18n/de_DE/messages/messages.json

```diff
@@ -44,6 +44,8 @@
   "bank_data_operator_account_owner": "Wikimedia e. V.",
   "bank_data_operator_bank_name": "SozialBank",
   "bank_data_transfer_code": "Verwendungszweck",
+  "bank_data_transfer_amount_label": "Betrag",
+  "bank_data_transfer_amount": "{formattedAmount}",
```

Comment:
You don't need this item as you can insert the formatted amount into the template directly.